frontend/app/templates/work_packages/tabs/activity.html

```diff
@@ -10,12 +10,14 @@
           <h3 class="activity-date"
               ng-if="currentDate !== (activities[$index-1].props.createdAt | date:'longDate')"
               ng-bind="currentDate"/>
-          <user-activity work-package="workPackage"
-                         activity="activity"
-                         activity-no="activityNo"
-                         is-initial="isInitial"
-                         input-element-id="'add-comment-text'">
-          </user-activity>
+            <activity-entry work-package="workPackage"
+                           activity="activity"
+                           activity-no="activityNo"
+                           is-initial="isInitial"
```

Comment:
> I don't think the second one would work, though, or can you mention a work package prior to its creation using the `#id` notation? Even if you mention some WP unintentionally, the created journal entry still would precede it.

This is my fear. That I could unintentionally reference it prior its creation and later when the WP is created it will be linked to that commit and thereby become the "initial" activity as of the current implementation.

If this was not possible (I don't know for sure), the current implementation (`activityNo == 1`) would be sufficient...